### PR TITLE
Config

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/service/config/ServoConfig.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.service.config;
+
+public class ServoConfig extends Config {
+
+  public Boolean autoDisable;
+  public String controller;
+  public Double currentOutputPos;
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/config/ServoConfig.java]: we can specify the current input postion. .we should not put output current position in the config.

File: src/main/java/org/myrobotlab/service/config/ServoConfig.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  public Boolean isBlocking;
+  
+  public String pin;
+  public Double rest;
+  public Double speed;
+
+  public Double sweepMax;
+  public Double sweepMin;
+  
+  public Double targetPos;
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/config/ServoConfig.java]: questionable if we care about targetPos

File: src/main/java/org/myrobotlab/service/config/ServoConfig.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+public class ServoConfig extends Config {
+
+  public Boolean autoDisable;
+  public String controller;
+  public Double currentOutputPos;
+  public Boolean enabled;
+  public Boolean idleDisabled;  
+  public Integer idleTimeout;
+  public Boolean isBlocking;
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/config/ServoConfig.java]: isn't this a detail of a movement?  we shouldn't have  isBlocking in the config.

Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/ServoConfig.java]: agree

File: src/main/java/org/myrobotlab/service/config/ServoConfig.java (hunk 2 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/ServoConfig.java]: It works out to be the value of "last input" ...
not sure - I'm going to take it out, if a "real encoder" existed it should be taking the value when starting there

**Pull request merged**